lib/private/validate-configured-input-values.js

```diff
@@ -45,6 +45,35 @@ module.exports = function validateConfiguredInputValues(machine){
         );
         return;
       }
+      // For inputs with string-type examples (excluding * and ===, which have special meaning), don't allow empty string as a value.
+      if (_.isString(inputDef.example) && inputDef.example !== '*' && inputDef.example !== '===' && machine._configuredInputs[inputName] === '') {
```

Comment:
LGTM except don't forget `example: '->'` (lamda).  For simplicity I think you could use `rttc.getDisplayType()` here on the exemplar -- e.g.:

![image](https://cloud.githubusercontent.com/assets/618009/25549401/23cbfbe6-2c37-11e7-95e3-802d0a319209.png)